app/presenters/hyrax/users_presenter.rb

```diff
@@ -0,0 +1,55 @@
+module Hyrax
+  class UsersPresenter
+    attr_reader :query, :authentication_key
+
+    def initialize(query:, authentication_key:)
+      @query = query
+      @authentication_key = authentication_key
+    end
+
+    # @return [Array] an array of Users
+    def users
+      @users = search(query)
+    end
+
+    def user_count
+      ::User.registered.without_system_accounts.count
+    end
+
+    def repository_admin_count
```

Comment:
This is a very expensive operation. It will iterate over all the users in the system.  This could be in the thousands. Can we count admin roles instead? I'm comfortable with the assumption that the system accounts will not be granted an admin role.